common-pipeline/operators/conditions-v3-schema.json

```diff
@@ -523,6 +553,16 @@
                 "op",
                 "values"
               ]
+            },
+            {
+              "properties": {
```

Comment:
This was added for filter conditions to only specify `op`, specifically for `dmSharedFields`

Reply:
yes that makes sense, i'll make the change